core/templates/pages/exploration-editor-page/statistics-tab/statistics-tab.directive.html

```diff
@@ -6,10 +6,10 @@ <h3>Exploration Completion Rate</h3>
         <pie-chart data="$ctrl.pieChartData"
                    options="$ctrl.COMPLETION_RATE_PIE_CHART_OPTIONS">
         </pie-chart>
-        <p style="text-align: center;"><span ng-if="$ctrl.numPassersby" style="font-weight: bold;" class="protractor-test-num-passersby"><[$ctrl.numPassersby]></span><span ng-if="!$ctrl.numPassersby" style="font-weight: bold;">0</span> learners browsed this exploration. They left before reaching the second card.</p>
+        <p class="oppia-num-passer-by-container"><span ng-if="$ctrl.numPassersby" class="oppia-num-passer-by protractor-test-num-passersby"><[$ctrl.numPassersby]></span><span ng-if="!$ctrl.numPassersby" class="oppia-num-passer-by">0</span> learners browsed this exploration. They left before reaching the second card.</p>
```

Comment:
```suggestion
        <p class="text-center"><span ng-if="$ctrl.numPassersby" class="oppia-num-passer-by protractor-test-num-passersby"><[$ctrl.numPassersby]></span><span ng-if="!$ctrl.numPassersby" class="oppia-num-passer-by">0</span> learners browsed this exploration. They left before reaching the second card.</p>
```

Ditto elsewhere in your changes.